.tests/integration/config.yaml

```diff
@@ -16,13 +16,18 @@ reference:
 
 bwa_mem:
   container: "docker://hydragenetics/bwa_mem:0.7.17"
-  extra: "-R '@RG\\tID:{sample}\\tSM:{sample}\\tPL:illumina\\tPU:{sample}' -v 1 "
 
 mark_duplicates:
   container: "docker://gmsuppsala/somatic:develop"
 
 samtools_index:
-  container: "docker://gmsuppsala/somatic:develop"
+  container: "docker://hydragenetics/common:0.0.1"
```

Comment:
If we use the default container, do we still need to specify that?

Reply:
maybe not! Let's decide on a common practice.

Reply:
yes please! 👍